developer/src/kmc-kmn/src/compiler/compiler.ts

```diff
@@ -65,8 +66,11 @@ export class Compiler {
       this.compileKeyboardFile = this.wasmModule.cwrap('kmcmp_Wasm_CompileKeyboardFile', 'boolean', ['string', 'string',
         'number', 'number', 'number', 'string']);
       this.setCompilerOptions = this.wasmModule.cwrap('kmcmp_Wasm_SetCompilerOptions', 'boolean', ['number']);
+      this._parseUnicodeSet = this.wasmModule.cwrap('kmcmp_Wasm_ParseUnicodeSet', 'number', [ 'string', 'number', 'number']);
```

Comment:
I think we should keep the member names consistent -- perhaps rename `compileKeyboardFile` and `setCompilerOptions` to `_compileKeyboardFile` and `_setCompilerOptions` as well?

Reply:
Except I think setCompilerOptions is called directly, whereas parseUnicodeSet has a shim JS function?

Reply:
OK. I guess `run` and `runCompiler` are the real API.

I'll prefix it with `wasm`.  so `wasmCompileKeyboardFile`